### PR TITLE
Expose new node limit settings for use with SET GLOBAL

### PR DESCRIPTION
**Pull request by mfussenegger**
## Summary of the changes / Why this improves CrateDB

Follow up to https://github.com/crate/crate/pull/11759

## Checklist

 - [x] Added an entry in `CHANGES.txt` for user facing changes
 - [x] Updated documentation & `sql_features` table for user facing changes
 - [x] Touched code is covered by tests
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

File: server/src/test/java/io/crate/integrationtests/InformationSchemaTest.java [hunk 1]
```diff
@@ -531,7 +531,7 @@ public void testTableConstraintsWithOrderBy() {
     @Test
     public void testDefaultColumns() {
         execute("select * from information_schema.columns order by table_schema, table_name");
-        assertEquals(858, response.rowCount());
+        assertEquals(863, response.rowCount());
```
Comment (seut) [inline server/src/test/java/io/crate/integrationtests/InformationSchemaTest.java:534]: mistake? or how is this related?

**Review by seut — APPROVED** [review]
Lgtm otherwise

File: server/src/test/java/io/crate/integrationtests/InformationSchemaTest.java (hunk 1 above)
Comment (mfussenegger) [reply server/src/test/java/io/crate/integrationtests/InformationSchemaTest.java:534]: The settings are exposed as columns - (also just force pushed another change because the doc-tests were failing)

Comment (seut) [reply server/src/test/java/io/crate/integrationtests/InformationSchemaTest.java:534]: ah by `sys.cluster.settings` right, sry.
